src/test/java/org/openhim/mediator/denormalization/CSDRequestActorTest.java

```diff
@@ -218,7 +218,7 @@ public void resolveFacilityIdentifier() throws Exception {
                 actor.tell(testMsg, getRef());
 
                 ResolveFacilityIdentifierResponse response = expectMsgClass(
-                        Duration.create(100, TimeUnit.MILLISECONDS),
+                        Duration.create(2000, TimeUnit.MILLISECONDS),
```

Comment:
Are these 2 second timeouts? Maybe we should make these 60s. I think 60s is fair for most timeouts.